idaes/config.py

```diff
@@ -218,6 +218,33 @@ def _new_idaes_config_block():
             },
         ),
     )
+    cfg.declare(
+        "properties",
+        pyomo.common.config.ConfigBlock(
+            implicit=False,
+            description="Physical property method configuration block",
+            doc="Physical property method configuration block",
```

Comment:
Mostly for clarification, but what are these used for? You might want to expand the `doc` attribute to give a bit more detail.

Reply:
There is only the setting for the parameter file directly.  I just added the property level figuring there could be other settings for other property methods at some point. I'll see if I can make this a little more clear.